### ISSUE TITLE
Instructions in WIKI for compiling on Mac

### ISSUE THREAD

**Issue by orsi** [snes9xgit/snes9x#665]
Currently there are only instructions on how to build on Windows with Visual Studio 2017. It would be great to have instructions also on how to build with Mac specifically, even if it is also with Visual Studio.

**Comment by MichaelBuckley** [comment]
Compiling on the Mac is very easy, and the steps are the same as most Mac projects. I'll add a Wiki page for it, but the steps are essentially:

- Download Xcode from Apple
- Open the macosx/Snes9x.xcodeproj file
- Click the run button at the top-left of the window.

**Comment by MichaelBuckley** [comment]
I have added Mac build instructions to the wiki.

I have also updated the Xcode project file so that debug builds no longer require code signing. If you were having trouble building before, pull down the latest commit and it should be fixed.

**Comment by orsi** [comment]
Figured I was just missing or looking over an XCode project file. I attempted trying to build from the terminal at first but I'll go the easier route.

Currently having trouble building the project in XCode. A couple errors I've run into:

`No account for team "S4YHUE5Y65". (Add a new account in the Accounts preference pane or verify that your accounts have valid credentials.)`

and

`dyld: Library not loaded...`

Had to add my Apple Id to Preferences -> Accounts and then tried switching the project Signing & Capabilities over to my personal account. Removes the previous errors but still doesn't build:

`
2020-09-11 14:31:04.690711-0400 Snes9x[19220:18037570] Metal API Validation Enabled
2020-09-11 14:31:04.995002-0400 Snes9x[19220:18037570] [plugin] AddInstanceForFactory: No factory registered for id <CFUUID 0x600000250520> F8BB1C28-BAE8-11D6-9C31-00039315CD46
2020-09-11 14:31:05.037209-0400 Snes9x[19220:18037570]  HALC_ShellDriverPlugIn::Open: Can't get a pointer to the Open routine
2020-09-11 14:31:05.037771-0400 Snes9x[19220:18037570]  HALC_ShellDriverPlugIn::Open: Can't get a pointer to the Open routine
Interval: 16ms, Frames: 512
2020-09-11 14:31:05.055088-0400 Snes9x[19220:18037570] [default] 0x10008eae6: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.055237-0400 Snes9x[19220:18037570] [default] 0x10008eae6: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.064816-0400 Snes9x[19220:18037570] [default] 0x10008ea3a: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.064914-0400 Snes9x[19220:18037570] [default] 0x10008ea3a: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.065216-0400 Snes9x[19220:18037570] [default] 0x10008ea35: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.065289-0400 Snes9x[19220:18037570] [default] 0x10008ea35: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.065586-0400 Snes9x[19220:18037570] [default] 0x10008ea3d: TCC deny IOHIDDeviceOpen
2020-09-11 14:31:05.065659-0400 Snes9x[19220:18037570] [default] 0x10008ea3d: TCC deny IOHIDDeviceOpen
`

**Comment by orsi** [comment]
Got it running. It was a combination of adding my Apple Id to XCode Accounts, and also removing Input Monitoring for Snes9x.

- Go to Apple Icon -> System Preferences -> Security & Privacy -> Input Monitoring
- Removing Snes9x entry
- Run in Xcode again

**Comment by MichaelBuckley** [comment]
Glad you were able to get it working. I tried to make it so that you wouldn't need to change the targets to use your account in Signing & Capibilities, since that configuration is shared between all users of the project. If you check that project change into git, I'd need to go change it back every time I built.

I just pushed another change that hopefully achieves this.

I will add the part about input monitoring to the wiki.
